src/config.ts

```diff
@@ -238,6 +238,19 @@ export const MEMPOOL_POLLING_INTERVAL_MS = +env.varOrDefault(
   '30000', // 30 seconds
 );
 
+//
+// AWS settings
+//
+//
+export const AWS_ACCESS_KEY_ID = env.varOrUndefined('AWS_ACCESS_KEY_ID');
+export const AWS_SECRET_ACCESS_KEY = env.varOrUndefined(
+  'AWS_SECRET_ACCESS_KEY',
+);
+export const AWS_REGION = env.varOrUndefined('AWS_REGION');
+export const AWS_ENDPOINT = env.varOrUndefined('AWS_ENDPOINT');
+export const AWS_S3_BUCKET = env.varOrUndefined('AWS_S3_BUCKET');
```

Comment:
These two don't have much flexibility if we need to use different buckets for different things. Might be ok for now though.

Reply:
Good point. We can implement multi bucket support in the future.